docs/concepts/search/searchable-properties/spans.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Span Properties
+description: "Learn more about searchable span properties."
+sidebar_order: 21
+---
+
+Spans are the underlying data element captured in Sentry SDKs to build a trace.
+
+You can search by span properties in the following [sentry.io](https://sentry.io) pages:
+
+- Traces - when searching for a trace containing spans with matching properties
+
+When searching on span properties within the **Traces** page, the search will return any trace that contains _one or more spans_ matching the supplied span properties filter.
+
+## Searchable Properties
+
+Below is a list of keys and tokens that can be used in the span search.
+
+<DefinitionList>
+
+### `action`
+
+The type of span action.
+
+- **Type:** string
+
+### `browser.name`
+
+The name of the browser that generated the span.
+
+- **Type:** string
+
+### `cache.hit`
+
+Whether a cache read has hit or missed the queried cache key. Values are `'true'` or `'false'`.
+
+- **Type:** string
+
+### `description`
+
+Parameterized and scrubbed description of the span.
+
+- **Type:** string
+
+### `device.class`
+
+Device class is a synthesized field calculated based off device info found in context such as model (for iOS devices), and device specs like processor_frequency (for Android devices).
+
+- **Type:** string
+
+### `environment`
+
+Refers to your code deployment naming convention. For example, `development`, `testing`, `staging` and so on. [Learn more](/product/sentry-basics/environments/). In some pages of [sentry.io](https://sentry.io/), you filter on environment using a dropdown.
+
+- **Type:** string
+
+### `file_extension`
+
+The file extension of a resource span.
+
+- **Type:** string
+
+### `http.decoded_response_content_length`
+
+The [decoded body size](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/decodedBodySize) of the resource.
+
+- **Type:** string
+
+### `http.response_content_length`
+
+The [encoded body size](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/encodedBodySize) of the resource.
+
+- **Type:** string
+
+### `http.response_transfer_size`
+
+The [total transfer size](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/transferSize) of the resource.
+
+- **Type:** string
+
+### `messaging.destination.name`
+
+The queue name or topic from which the message is consumed from or published to.
+
+- **Type:** string
+
+### `messaging.message.id`
+
+The unique ID of a message from the queue or messaging system.
+
+- **Type:** string
+
+### `op`
+
+Span operation
+
+- **Type:** string
+
+### `os.name`
+
+Name of the operating system
+
+- **Type:** string
+
+### `platform`
+
+Name of the platform. This is only a metrics property for [valid platforms](https://github.com/getsentry/relay/blob/9d0d2dc54233d46043a369e1dacf7db900953c14/relay-general/src/protocol/constants.rs#L1-L21), defaulting to `other`.
```

Comment:
Hmm this seems really brittle since the line numbers & file location of our code could change at any moment, probably worth just dumping the information from that constants file here IMO

Reply:
Updated to an inline list of supported platforms, and updated the wording to drop "metrics." Made the same change in the events doc as well.